UnityProject/Assets/Scripts/Core/Chat/Chat.cs

```diff
@@ -125,6 +125,22 @@ public static void AddChatMsgToChat(ConnectedPlayer sentByPlayer, string message
 			VoiceLevel = loudness
 		};
 
+		foreach (ItemSlot slot in sentByPlayer.Script.DynamicItemStorage.GetNamedItemSlots(NamedSlot.ear))
+		{
+			Headset headset = slot.Item.gameObject.GetComponent<Headset>().OrNull();
+			if(headset != null)
+			{
+				if (headset.LoudSpeakOn && channels.HasFlag(ChatChannel.Common) ||
+				    channels.HasFlag(ChatChannel.Command) || channels.HasFlag(ChatChannel.Security)
+				    || channels.HasFlag(ChatChannel.Engineering) || channels.HasFlag(ChatChannel.Medical)
+				    || channels.HasFlag(ChatChannel.Science)
+				    || channels.HasFlag(ChatChannel.Syndicate) || channels.HasFlag(ChatChannel.Supply))
+				{
+					chatEvent.VoiceLevel = headset.LoudspeakLevel;
+				}
+			}
+		}
```

Comment:
DynamicItemStorage needs to be null checked as ghost, Ai and blob dont have storage

Reply:
and Item can be null

Reply:
Done.

Reply:
Alright now it's done done